pom.xml

```diff
@@ -351,7 +351,6 @@
 
                 <!-- <module>ethereum</module> --> <!-- JAVA-6001 -->
                 <!-- <module>gradle-modules</module> --> <!-- Not a maven project -->
-                <module>gradle-modules/gradle/maven-to-gradle</module>
```

Comment:
you need to remove from default-first profile too

Reply:
Done